source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -5,12 +5,140 @@
+representation.
+
+It can also provide an abilities to filter or normalize :term:`feature` values
+being extracted or recover missing values:
+
+- It is not a rare situation when only a subset of features can be measured at a
+  given moment. In this case, the non-measured features in the feature vector
+  become blank, or missing. Special statistical techniques enable recovery
+  (emulation) of missing values.
+
+- When a given measurement introduces an :term:`outlier` or when an observed
+  object can produce an :term:`outlier` by nature, special statistical techniques
+  enable detection of outliers and recovery of the abnormal data.
+
+Data are not always fit the local memory, especially when processing with
+accelerators. Data source provides an interface to load data by batches and
+orchestrates its efficient extraction directly into local memory of the device.
+Therefore data source enables complex scenarios of data analytics, like
+:term:`online computations <Online mode>`.
+
+Table builder
+-------------
+
+Some :term:`table` objects require complex procedures to create them. To make
+easy-of-use table creation from data of various :term:`data formats <Data
+format>` or other objects of different types, |dal_short_name| introduces a
+special :term:`builder` concept for table types. See :ref:`table builders
+<table_builders>` section for details.
+
+Accessor
+--------
+
+There are a lot of :term:`data formats <Data format>` used in :term:`datasets
+<Dataset>` to store the data. |dal_short_name| defines different types of
+:term:`tables <Table>`, table builders and other objects to support different
+logical types of data and various :term:`data formats <Data format>`. However,
+all these objects have different logic to access their underlying data.
+
+Accessor allows:
+
+- to make uniform access to the data from various sets of objects of different
+  types.
+
+- to perform efficient access to the data without exposing implementation
+  details of a data container.
+
+- to give an ability to focus on the data acquisition in the desired :term:`data
+  format` with a wanted set of operations over the data.
+
+See :ref:`accessors` section for details.
+
+Relations between concepts
+==========================
+
+Typical workflow with data in |dal_short_name| consists of next steps:
+
+1. Select an appropriate data source for extracting out-of-memory data, or
+   select a table builder to manually create a table object.
+
+2. Create :term:`table` object - a numerical representation of data, with use
+   of selected data source, table builder or table objetc constructor directly.
+
+3. Run a selected :ref:`algorithm <Algorithms>` with created table object.
+
+4. Get algorithm :ref:`results <Result>` and access their data directly or using
+   selected accessor.
+
+.. image:: _static/typical_data_flow.png
+  :width: 800
+  :alt: Typical data flow
+
+Next sequence diagram briefly shows dependencies between table, builder, and
```

Comment:
This part of the info logically comes after sections with table/builder/accessor